ajax/libs/algebra.js/0.2.5/algebra.js

```diff
@@ -0,0 +1,37 @@
+var Fraction = require('./src/fractions');
+var Expression = require('./src/expressions').Expression;
+var Equation = require('./src/equations');
+var Parser = require('./src/parser');
```

Comment:
This used to work; I believe the minify tool I was using took care of it somehow. This seems to have changed over the last few months (see https://github.com/nicolewhite/algebra.js/issues/56), but I don't know where or how. Any help is appreciated.